grails-app/services/au/org/emii/gogoduck/job/JobExecutorService.groovy

```diff
@@ -1,11 +1,82 @@
 package au.org.emii.gogoduck.job
 
+import au.org.emii.gogoduck.worker.Worker
+
+import java.util.concurrent.LinkedBlockingQueue
+import java.util.concurrent.BlockingQueue
+
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
 class JobExecutorService {
 
+    static BlockingQueue<Map> JOB_QUEUE
```

Comment:
I think having a limit here is a good idea. However I understand if it exceeds the scope as it'll require us to return error codes and all sort of other things, so OK to be left like that as well - but should go on the backlog.